Header header
  uint32 seq
  time stamp
  string frame_id
float64[7] joint_position
float64[7] joint_speed
float64[7] joint_jerk
float64[7] joint_tau
float64[7] joint_initial_tau
float64[7] joint_coriolis
float64[7] joint_gravity